bin/seed_test_users.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+numberExpression='^[0-9]+$'
+databasePath=var/user-lifecycle.sqlite
+
+if ! [[ $1 =~ $numberExpression ]] ; then
+   echo "Usage: The first argument must be a number. Indicating the number of identities you'd like to seed to the last_login table" >&2; exit 1
+fi
+
+if ! [[ $2 =~ $numberExpression ]] ; then
+   echo "Usage: The second argument must be a number. Indicating the number of days since last login for all users" >&2; exit 1
+fi
+
+if ! [ -x "$(command -v sqlite3)" ]; then
+    echo "The SQLite binary is not available"
+fi
+if ! [ -x "$(command -v rig)" ]; then
+    echo "The rig binary is not available, Rig is used to create a random username"
+fi
+
+if ! [ -f "${databasePath}" ]; then
+    echo "The SQLite database was not found at '${databasePath}'"
+fi
+
+echo "Adding ${1} users to the last_login test table.."
+for ((i = 1 ; i <= $1 ; i++)) do
+    randomUserName=`rig | head -n 1 |  tr ' ' _ | tr A-Z a-z`
+    timeLastLogin=`date --rfc-3339=seconds -d "-${2} days"`
```

Comment:
The database that we use in production has the format "YYYY-mm-dd":
MariaDB [log_logins_prd]> select * from last_login limit 1;
+------------------------------------+------------+
| userid                             | lastseen   |
+------------------------------------+------------+
| urn:collab:person:example.com:1234 | 2019-02-15 |
+------------------------------------+------------+

Reply:
Cool, changed the format to `date`